php/oop/home-work/UserValidation/UserValidation.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class User{
+
+    public function __construct(public string $email)
```

Comment:
Good job
It seems there is an issue, The `$email` property is public, It's possible to change it manually without passing validation rules. Could you fix it, please?